base-plugin/src/integTest/groovy/com/github/jrubygradle/JRubyPrepareGemsIntegrationSpec.groovy

```diff
@@ -51,6 +52,48 @@ class JRubyPrepareGemsIntegrationSpec extends IntegrationSpecification {
         new File(projectDir, "gems/rack-1.6.11").exists()
     }
 
+    @IgnoreIf({ IntegrationSpecification.OFFLINE })
```

Comment:
This verifies that the Gradle 5.x dependency locking feature can be used to control GEM dependencies too.